src/utils/utils.cairo

```diff
@@ -210,6 +210,34 @@ namespace Helpers {
         return current;
     }
 
+    // @notice This function is used to convert a sequence of 20 bytes big-endian
+    // to felt.
+    // @param val: pointer to the first byte of the 20.
+    // @return res: felt representation of the given input in bytes20.
+    func bytes20_to_felt(val: felt*) -> felt {
```

Comment:
I think that it's better to create a const with `dw` and use it, but not sure tbh. We'd need to make a unit test and run with `--profile-cairo` to make sure

Reply:
59 steps to 81 with a dw table

Reply:
add unit test